site/docs/install-ubuntu.md

```diff
@@ -3,25 +3,29 @@ layout: documentation
 title: Installing Bazel on Ubuntu
 ---
 
-# <a name="ubuntu"></a>Install Bazel on Ubuntu
+# <a name="ubuntu"></a>Install Bazel on Linux
 
-Supported Ubuntu Linux platforms:
+Supported Linux platforms:
 
-*   16.04 (LTS)
-*   14.04 (LTS)
+*  Ubuntu
+   *   16.04 (LTS)
+   *   14.04 (LTS)
+*  CentOS 6.7
 
-Install Bazel on Ubuntu using one of the following methods:
+Install Bazel on Linux using one of the following methods:
 
-*   [Use our custom APT repository (recommended)](#install-on-ubuntu)
-*   [Use the binary installer](#install-with-installer-ubuntu)
+*   [Use our custom APT repository (Ubuntu only)](#install-on-ubuntu)
+*   [Use the binary installer](#install-with-installer-linux)
```

Comment:
I don't see an installer for CentOS -- can you only compile from source to install on that OS?